pom.xml

```diff
@@ -180,12 +180,12 @@
                     <imageManagement>
                         <image>
                             <repository>${dockerHubPublic}/cafapi/opensuse-jre11</repository>
-                            <tag>4.0.0</tag>
-                            <digest>sha256:03c2b7fe78f3db1831a492ca341266eca7efb92f1f4968939308b9d038aba97c</digest>
+                            <tag>4.0.1</tag>
+                            <digest>sha256:6fc1394f496993ec4b693fd6ab947e610a6f1f55629b23ce97fd07399f509a38</digest>
                         </image>
                         <image>
                             <repository>${dockerHubPublic}/opensuse/leap</repository>
-                            <tag>15.6</tag>
+                            <tag>latest</tag>
```

Comment:
Also updated this as looks like a mistake in https://github.com/CAFapi/opensuse-tomcat-jre11-image/pull/70